tests/posix/README.md

```diff
@@ -0,0 +1,17 @@
+Test for POSIX Extensions:
+==========================
+
+This directory contains tests for the Open Enclave POSIX extensions, including:
+
+- dup - tests the dup() function.
+- fs - file system tests.
+- hostfs - host file system tests.
+- ids - tests the getuid(), getgid(), etc.
+- poller - tests the select() function.
+- polling - event polling tests.
+- resolver - tests for getnameinfo() and getaddrinfo().
+- sendmsg - tests for sendmsg() and recvmsg() over sockets.
+- send - tests for send() and recv() over sockets.
+- sgxfs - tests for the Intel protected file system.
+- socketpair - tests for the socketpair() function.
+- socket - tests for host sockets.
```

Comment:
The contents of this readme file does not seem to match those tests included in this PR.
Need to add:
 cpio platform

Need to remove
polling, sgxfs, send

Reply:
Thanks! Fixed.